cluster/etcd.go

```diff
@@ -17,16 +18,52 @@ import (
 
 func (c *Cluster) SnapshotEtcd(ctx context.Context, snapshotName string) error {
 	backupImage := c.getBackupImage()
+	containerTimeout := DefaultEtcdBackupConfigTimeout
+	if c.Services.Etcd.BackupConfig != nil && c.Services.Etcd.BackupConfig.Timeout > 0 {
+		containerTimeout = c.Services.Etcd.BackupConfig.Timeout
+	}
+
+	// store first error message
+	var snapshotErr error
+	snapshotFailures := 0
+	s3Failures := 0
```

Comment:
Nit: This specifies the count of failures to upload an etcd snapshot to s3. At first glance I thought, what kind of s3 failure is this? Rename to `s3UploadFailures`